src/edu/csus/ecs/pc2/core/imports/LoadAccounts.java

```diff
@@ -397,4 +534,172 @@ public static void updateAccountsFromLoadAccountsFile(IInternalContest contest,
+            externalIdColumn = -1;
+            permDisplayColumn = -1;
+            permLoginColumn = -1;
+            permPasswordColumn = -1;
+            scoreAdjustmentColumn = -1;
+            for (int i = 0; i < columns.length; i++) {
+                
+                if (Constants.SITE_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    siteColumn = i;
+                }
+                if (Constants.ACCOUNT_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    accountColumn = i;
+                }
+                if (Constants.DISPLAYNAME_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    displayNameColumn = i;
+                }
+                if (Constants.PASSWORD_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    passwordColumn = i;
+                }
+                if (Constants.GROUP_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    groupColumn = i;
+                }
+                if (Constants.PERMDISPLAY_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    permDisplayColumn = i;
+                }
+                if (Constants.PERMLOGIN_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    permLoginColumn = i;
+                }
+                if (Constants.EXTERNALID_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    externalIdColumn = i;
+                }
+                if (Constants.ALIAS_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    aliasColumn = i;
+                }
+                if (Constants.PERMPASSWORD_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    permPasswordColumn = i;
+                }
+                if (Constants.LONGSCHOOLNAME_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    longSchoolNameColumnn = i;
+                }
+                if (Constants.SHORTSCHOOLNAME_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    shortSchoolNameColumn = i;
+                }
+                if (Constants.COUNTRY_CODE_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    countryCodeColumn = i;
+                }
+                if (Constants.TEAMNAME_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    teamNameColumn = i;
+                }
+                if (Constants.SCORING_ADJUSTMENT_COLUMN_NAME.equalsIgnoreCase(columns[i])) {
+                    scoreAdjustmentColumn = i;
+                }
+            }
+            if (accountColumn == -1 || siteColumn == -1) {
+                String msg = "1st line should be the row headers (account and site are required)";
+                in.close();
+                throw new IllegalTSVFormatException(msg);
+            }
+        }
+        line = in.readLine();
+        lineCount++;
+        while (line != null) {
+            try {
+                // skip comments & line blanks
+                if (line.startsWith("#") || line.equals("")) {
```

Comment:
It's interesting that THIS test (unlike the `while` loop commented on above), _does_ check for the empty string ("") -- but it _doesn't_ check for a null string...  Seems like it should?

Reply:
At that point it is impossible for line == null, right ?

Reply:
re: checking for null string, isn't that done in the if at line 675?

Reply:
John C indicated that this is resolved.